lib/linguist/languages.yml

```diff
@@ -2543,6 +2543,15 @@ MiniD:
   tm_scope: none
   ace_mode: text
   language_id: 231
+MiniZinc:
+  type: programming
+  color: "#1295e6"
+  extensions:
+  - ".mzn"
+  - ".dzn"
```

Comment:
Yes, `.dzn` files provide data for the constraints defined in `.mzn` files. The grammar for `.dzn` files is a subset of the grammar of `.mzn` files and so I think using the same syntax is appropriate.